pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -1417,6 +1417,10 @@ type GatewaySpec struct {
 type ZoneSpec struct {
 	// RGW Zone the Object Store is in
 	Name string `json:"name"`
+	// For the multisite configuration the endpoints need to be reachable,
```

Comment:
The comment says "endpointS" but it's just a string, is it expected? Can we have more than one endpoint?
If yes, then should we use `v1.Subsets` for the `v1.Endpoints` configuration?

Reply:
The multisite configuration for zonegroup contains the list of endpoints which is populated from different cephobjectstore. The field represents a custom endpoint for the ceph object store which is a single entity.  So the string is sufficient IMO

Reply:
I agree here that when the API we provide is singular, it's confusing for the description to be plural.

Reply:
Agree with Blaine, I think we just need to change `endpoints need to be reachable` to `**endpoint** needs to be reachable`